src/tiledb/cloud/sql/tiledb_connection.py

```diff
@@ -0,0 +1,23 @@
+from tiledb.cloud.sql.db_api_exceptions import NotSupportedError
+from tiledb.cloud.sql.tiledb_cursor import Cursor
+
+
+class TileDBConnection:
+    def cursor(self):
+        return Cursor()
+
+    def rollback(self):
+        raise NotSupportedError("Operation not supported")
```

Comment:
per the PEP](https://peps.python.org/pep-0249/#id51) it is recommended to [have this not be present](https://peps.python.org/pep-0249/#id51). (It might be worth adding a comment under `commit` to the effect of “no `rollback` method due to no transaction support”